### ISSUE TITLE
modular window icon / button

### ISSUE THREAD

**Issue by Reaper10** [swesterfeld/beast#14]
Thisn't issue I just have made a modular window icon / button
!
![rack icon full colour](https://user-images.githubusercontent.com/8137561/61488582-3a3bb900-a96e-11e9-8bde-8dfc16b76ef9.png)
https://github.com/Reaper10/ui-kit-for-the-beast-daw/tree/master/window%20icons

**Comment by swesterfeld** [comment]
Right now, I don't believe we can accept any icon contribution. Basically, we have icons for beast-gtk (which we will throw away at some point in the future), and the new electron based ebeast UI is not at a point where icon design (or other design) is necessary.

If I remember correctly, the only plugin that would need a new icon would be the spectmorph branch https://github.com/tim-janik/beast/pull/30 - you'd have to look at other beast plugin icons to get a feeling how this should look. But this is plugin has not been merged yet.

So basically you can contribute
- make music using beast (this is by far the most important input: especially if you publish the .bse files)
- produce instruments
- produce effects
- produce .bsewave files
- code
- documentation
- talk about beast on youtube
- blog about beast
- testing
- ... (this list may not be complete)

**Comment by Reaper10** [comment]
new specmorph plugin icon I did #83

**Comment by Reaper10** [comment]
new icon
https://github.com/Reaper10/ui-kit-for-the-beast-daw/blob/master/beast%20icons/rack%20icon%20full%20colour.png
